libraries/apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo3/network/ws/AppSyncWsProtocol.kt

```diff
@@ -17,18 +17,22 @@ import kotlinx.coroutines.withTimeout
  * A [WsProtocol] for https://docs.aws.amazon.com/appsync/latest/devguide/real-time-websocket-client.html
  */
 class AppSyncWsProtocol(
-    private val authorization: Map<String, Any?>,
     private val connectionAcknowledgeTimeoutMs: Long,
+    private val connectionPayload: suspend () -> Map<String, Any?>? = { null },
```

Comment:
Looks like the compilation fails because the constructor is referenced by the Factory below

Reply:
woops, my bad. I was cutting/pasting changes because my editor wanted to change all the code styling.